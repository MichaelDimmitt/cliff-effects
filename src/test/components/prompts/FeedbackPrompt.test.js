import React from 'react';
import { mount } from 'enzyme';


import FeedbackPrompt from '../../../components/prompts/FeedbackPrompt';

test('A feedback prompt should match the snapshot', () => {
  let isBlocking = true;

  let promptData = {
    header:    'Test prompt data',
    leaveText: 'Reset',
  };

  const wrapper = mount(
    <FeedbackPrompt
      { ...promptData }
      isBlocking={ isBlocking }
      openFeedback={ jest.fn() } />
  );
  expect(wrapper).toMatchSnapshot();
});
