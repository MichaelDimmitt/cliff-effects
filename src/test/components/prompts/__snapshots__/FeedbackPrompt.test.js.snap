// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A feedback prompt should match the snapshot 1`] = `
<FeedbackPrompt
  header="Test prompt data"
  isBlocking={true}
  leaveText="Reset"
  openFeedback={[Function]}
>
  <Modal
    closeOnDimmerClick={true}
    closeOnDocumentClick={false}
    dimmer={true}
    eventPool="Modal"
    mountNode={null}
  >
    <Portal
      className="ui page modals dimmer transition visible active"
      closeOnDocumentClick={false}
      closeOnEscape={true}
      closeOnRootNodeClick={true}
      eventPool="Modal"
      mountNode={<body />}
      onClose={[Function]}
      onMount={[Function]}
      onOpen={[Function]}
      onUnmount={[Function]}
      openOnTriggerClick={true}
    />
  </Modal>
</FeedbackPrompt>
`;
